src/app/CommandSender.cpp

```diff
@@ -82,6 +82,29 @@ CHIP_ERROR CommandSender::SendCommandRequest(const SessionHandle & session, Syst
     return SendInvokeRequest();
 }
 
+CHIP_ERROR CommandSender::SendGroupCommandRequest(const SessionHandle & session, System::Clock::Timeout timeout)
+{
+    CHIP_ERROR err = CHIP_NO_ERROR;
+
+    VerifyOrExit(mState == State::AddedCommand, err = CHIP_ERROR_INCORRECT_STATE);
```

Comment:
Why are we using the `exit:` pattern when there is no actual code in the exit block?  Should just use `VerifyOrReturnError`, etc...

Reply:
did the switch for ReturnError.